src/View/SSTemplateParser.peg

```diff
@@ -405,6 +406,14 @@ class SSTemplateParser extends Parser implements TemplateParser
 
     QuotedString: q:/['"]/   String:/ (\\\\ | \\. | [^$q\\])* /   '$q'
 
+    # Booleans
+
+    Boolean: /(true|false)/
+
+    # Integers and floats
+
+    IntOrFloat: / (\d+(\.\d+)?) /
```

Comment:
Doh. This needs to be updated to support passing things like `.5`, `-10` etc. There are other expressions that are valid numbers in PHP (exponents, hexadecimal, binary), not sure if the effort/reward will be worth it for those - I’ll look into it